java_servlets/src/main/java/getNameServlet.java

```diff
@@ -0,0 +1,18 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/get")
+public class getNameServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        PrintWriter pw = response.getWriter();
+
+        pw.println("<html><big>The name is " + request.getParameter("name") + "</big></html>");
+
+        pw.close();
```

Comment:
Actually you don't need to close the `PrintWriter` here. You have just retrieved it from the getter, not created. The generic principle is "you close what you have yourself opened, but  you don't close what you haven't opened"

Reply:
Got it. Fixed all mistakes. Please check new version